.github/workflows/integrate.yaml

```diff
@@ -11,8 +11,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        juju: [2.8/stable]
-        microk8s: [latest/stable]
+        juju: [2.9/stable]
+        microk8s: [1.20/stable]
```

Comment:
This seems to have fixed that credentials issue that we had in the tests. Do we have a bug for what the issue was on microk8s `latest/stable`?

Reply:
https://bugs.launchpad.net/juju/+bug/1937282